var/spack/repos/builtin/packages/py-fenics-fiat/package.py

```diff
@@ -23,7 +23,6 @@ class PyFenicsFiat(PythonPackage):
 
     license("LGPL-3.0-or-later")
 
-    version("master", branch="master")
```

Comment:
We don't normally remove versions without first deprecating them

Reply:
Good point. Adding it back in https://github.com/spack/spack/pull/42182/commits/187f4792ae17728a237dfd0396211e43e85eb9fb